File: atv_classe1/java/python/ponto.py
class Ponto():
    def __init__(self, x = 0, y = 0):
        self.x = x
        self. y = y

    def set_x(self, x: int):
        self.x = x

    def get_x(self):
        return self.x

    def set_y(self, y: int):
        self.y = y

    def get_y(self):
        return self.y

    def to_string(self):
        return f"Pontos - X: {self.x}, Y:{self.y}"

File: atv_classe1/java/Ponto.java
package atv_classe1.java;

public class Ponto {
    protected int x;
    protected int y;

    public Ponto(int x, int y){
        this.x = x;
        this.y = y;
    }

    public Ponto() {
        this(0, 0);
    }

    public Ponto(int x) {
        this(x, 0);
    }

    public int getX(){
        return this.x;
    }

    public int gety(){
        return this.y;
    }

    public void setX(int x){
        this.x = x;
    }

    public void setY(int y){
        this.y = y;
    }

    public String toString(){
        return String.format("X: %d, Y: %d", x, y);
    }

}
